### PR TITLE
Docker: use dedicated db for telemetry

### PR DESCRIPTION
**Pull request by stsewd**
Ref https://github.com/readthedocs/readthedocs.org/pull/9113

### CODE REVIEW

File: dockerfiles/entrypoints/postgres.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+-- Initialization script.
+-- https://github.com/docker-library/docs/blob/master/postgres/README.md#initialization-scripts
```
Comment (stsewd) [inline dockerfiles/entrypoints/postgres.sql:2]: this is only executed the first time the container is created, if you already have one and don't want to re-create it, just create the db manually from the shell.

Comment (ericholscher) [reply dockerfiles/entrypoints/postgres.sql:2]: This comment should be in the code :)

**Review by ericholscher — APPROVED** [review]
Looks simple enough.

**Comment by humitos** [comment]
Why this is not required for `docs_db`?

**Comment by stsewd** [comment]
> Why this is not required for `docs_db`?

what thing?

**Comment by humitos** [comment]
The sql script with the `CREATE DATABASE` sentence.

**Comment by stsewd** [comment]
@humitos the one you declare here is created automatically, but you can't declare more than one

https://github.com/readthedocs/common/blob/baa26aa89229b0a59a9507eb5992e559d37775b1/dockerfiles/docker-compose.yml#L144

**Pull request merged**